.gitignore

```diff
@@ -2,15 +2,6 @@
 /dist/
 /storybook-static/
 
-# Yarn stuff; we're not using PnP/Zero installs
-.pnp.*
```

Comment:
@olemartinorg  The `.pnp.*` entry should probably still be kept?

Reply:
Right, yes, adding that one back. I moved a bit too quickly there!